### PR TITLE
docs: add traffic-split plugin usage

### PR DESCRIPTION
**Pull request by AlinsRan**
### Type of change:

- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches
- [x] Documentation
- [ ] Refactor
- [ ] Chore
- [ ] CI/CD or Tests

### Pre-submission checklist:

- [ ] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

File: docs/en/latest/plugins/traffic-split.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+```yaml
+backends:
+- serviceName: httpbin1
+  servicePort: 80 
+- serviceName: httpbin2
+  servicePort: 80
+plugins:
+- name: traffic-split # Not working, this plugin is invalid and cannot be used with multiple backend.
+  enable: true
+```
```
Comment (tao12345666333) [inline docs/en/latest/plugins/traffic-split.md:56]: I think it's better to give the full configuration instead of just partial snippets.

**Review by tao12345666333 — COMMENTED** [review]
I think it would be better to add a summary

File: docs/en/latest/plugins/traffic-split.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+#
+-->
+
+This guide describes how to use the traffic-split plugin in APISIX Ingress.
+
+The [traffic-split](https://apisix.apache.org/docs/apisix/plugins/traffic-split) plugin can be used to dynamically route part of traffic to various upstream services.
+
+:::info IMPORTANT
+
+If multiple services are declared in ApisixRoute `http[].backends`, it cannot be used with traffic-split.
```
Comment (Gallardot) [inline docs/en/latest/plugins/traffic-split.md:30]: Maybe this document could also be about traffic splitting using `http[].upstreams`?

**Review by lingsamuel — APPROVED** [review]

File: docs/en/latest/plugins/traffic-split.md (hunk 2 above)
Comment (AlinsRan) [reply docs/en/latest/plugins/traffic-split.md:30]: Do you mean external services?
As far as I know, http []. upstreams can only be used as an external service at present.
https://github.com/apache/apisix-ingress-controller/blob/1.6.0/pkg/providers/apisix/translation/apisix_route.go#L385-L386

Comment (Gallardot) [reply docs/en/latest/plugins/traffic-split.md:30]: Yes, instructions for using the traffic split plugin on the external service. Hope to add to that.

**Review by lingsamuel — APPROVED** [review]

**Review by tao12345666333 — APPROVED** [review]

**Pull request merged**